<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../바탕화면/image/logo.png</normaloff>../../바탕화면/image/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTextBrowser" name="Ttotalprice">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_pay">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>530</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);</string>
   </property>
   <property name="text">
    <string>결제하기</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tab1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>400</width>
     <height>341</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="cate1">
    <attribute name="title">
     <string>김밥류</string>
    </attribute>
    <widget class="QPushButton" name="Btn_cate1_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);</string>
     </property>
     <property name="text">
      <string>돈까스김밥</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/donkkas_1.png</normaloff>image/donkkas_1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_cate1_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);</string>
     </property>
     <property name="text">
      <string>참치 김밥</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/chamchi.png</normaloff>image/chamchi.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_cate1_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);</string>
     </property>
     <property name="text">
      <string>원조 김밥</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/kimbap_1.png</normaloff>image/kimbap_1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="cate2">
    <attribute name="title">
     <string>분식류</string>
    </attribute>
    <widget class="QPushButton" name="Btn_cate2_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);</string>
     </property>
     <property name="text">
      <string>떡볶이</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/ddeok.png</normaloff>image/ddeok.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_cate2_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);</string>
     </property>
     <property name="text">
      <string>라면</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/ramyeon.png</normaloff>image/ramyeon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_cate2_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);</string>
     </property>
     <property name="text">
      <string>우동</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/woodong.png</normaloff>image/woodong.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_cate2_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);</string>
     </property>
     <property name="text">
      <string>만두</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/mandoo.png</normaloff>image/mandoo.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="cate3">
    <attribute name="title">
     <string>식사류</string>
    </attribute>
    <widget class="QPushButton" name="Btn_cate3_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);</string>
     </property>
     <property name="text">
      <string>제육덮밥</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/pook_1.png</normaloff>image/pook_1.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_cate3_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);</string>
     </property>
     <property name="text">
      <string>김치찌개</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/kimchizzi.png</normaloff>image/kimchizzi.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_cate3_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);</string>
     </property>
     <property name="text">
      <string>돈까스</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/donkkas.png</normaloff>image/donkkas.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Btn_cate3_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 245, 244);</string>
     </property>
     <property name="text">
      <string>비빔밥</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>image/bibimbap.png</normaloff>image/bibimbap.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>총 금액</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_home">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);</string>
   </property>
   <property name="text">
    <string>처음으로</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_manage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(223, 113, 20, 0.0)



</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon2.png</normaloff>image/icon2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>85</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>메뉴</string>
    </property>
    <property name="background">
     <color>
      <red>246</red>
      <green>245</green>
      <blue>244</blue>
     </color>
    </property>
   </column>
   <column>
    <property name="text">
     <string>옵션</string>
    </property>
    <property name="background">
     <color>
      <red>246</red>
      <green>245</green>
      <blue>244</blue>
     </color>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>411</width>
     <height>611</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_del">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>빼기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap>image/logo2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>Ttotalprice</zorder>
  <zorder>Btn_pay</zorder>
  <zorder>label</zorder>
  <zorder>Btn_home</zorder>
  <zorder>tableWidget</zorder>
  <zorder>Btn_del</zorder>
  <zorder>label_2</zorder>
  <zorder>tab1</zorder>
  <zorder>Btn_manage</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
